src/schema/notifications.ts

```diff
@@ -126,10 +156,17 @@ export class ZimbraNotifications {
 						...item
 					}
 				});
+				if (sortBy) {
```

Comment:
if there is no sortyBy attribute then we will not concat new contact with search response, which is a bug

Reply:
as per my understanding when user deletes the contactgroup from list then in that case sortBy value is coming undefined.
so, that's why i put check here
while inserting into list we will always get the sortBy value

Reply:
is there any possibility while inserting we will get no sortBy value ?

if yes, then I will remove the check.

Reply:
don't just look at current scenarios, in future someone introduces a call without sortBy value then there will be a bug introduced unknowingly

Reply:
ok will remove the check